Misc/NEWS.d/next/Library/2018-08-20-16-48-32.bpo-34441._zx9lU.rst

```diff
@@ -1,2 +1,3 @@
-Fix crash when an ``ABC``-derived class with non-callable ``__subclasses__`` is passed as the
-second argument to ``issubclass()``.
+Fix crash when an ``ABC``-derived class with non-callable ``__subclasses__``
```

Comment:
Non-callable `__subclasses__` is not the only cause of a crash.

Reply:
You're right. I've also updated the PR name.